common/model-views.cpp

```diff
@@ -696,9 +731,17 @@ namespace rs2
                                 {
                                     set_option(opt, new_value, error_message);
                                 }
-
                                 edit_mode = false;
                             }
+                            else if (should_be_displayed_in_cm)
+                            {
+                                std::string buff_str(buff);
+                                float buff_float = std::stof(buff_str);
+                                buff_float /= 100.f;
+                                std::string buff_str_cm(std::to_string(buff_float));
+                                memset(buff, 0, TEXT_BUFF_SIZE);
+                                strcpy(buff, buff_str_cm.c_str());
+                            }
```

Comment:
Duplicated in lines 710-718. Make it a function call

Reply:
Done as lambda